package.json

```diff
@@ -1,56 +1,50 @@
 {
   "name": "eslint-config-molindo",
-  "version": "7.0.0",
+  "version": "8.0.0",
   "description": "Molindo ESLint config that implements our styleguide and helps to catch errors.",
   "repository": "https://github.com/molindo/eslint-config-molindo",
   "author": "Jan Amann <jam@molindo.at>",
+  "type": "module",
   "license": "MIT",
+  "main": "src/index.js",
   "keywords": [
     "eslint",
     "eslintconfig"
   ],
   "scripts": {
-    "lint": "eslint .",
-    "test": "jest",
-    "prepublish": "npm run lint && npm run test"
+    "lint": "eslint src && prettier src --check",
+    "prepublish": "npm run lint"
   },
   "files": [
-    "setupPlugins.js",
-    "javascript.js",
-    "typescript.js",
-    "config.js",
-    "css-modules.js",
-    "tailwind.js",
-    "react.js",
-    "jest.js",
-    "cypress.js",
+    "src",
+    ".prettierrc.json",
     "tsconfig.json"
   ],
   "devDependencies": {
-    "eslint": "^8.45.0",
-    "jest": "^29.6.2",
-    "typescript": "5.1.6"
+    "eslint": "^9.0.0",
+    "prettier": "^3.3.0",
+    "tailwindcss": "^3.4.13",
+    "typescript": "^5.1.6"
   },
   "peerDependencies": {
-    "eslint": "^8.0.0"
+    "eslint": "^9.0.0"
   },
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "^6.2.0",
-    "@typescript-eslint/parser": "^6.2.0",
+    "@eslint/js": "^9.11.1",
+    "@typescript-eslint/utils": "^8.0.0",
+    "@vitest/eslint-plugin": "^1.1.7",
     "confusing-browser-globals": "^1.0.11",
+    "eslint-import-resolver-typescript": "^3.6.3",
     "eslint-plugin-css-modules": "^2.11.0",
-    "eslint-plugin-import": "^2.27.5",
-    "eslint-plugin-jest": "^27.2.3",
-    "eslint-plugin-jsx-a11y": "^6.7.1",
-    "eslint-plugin-prettier": "^5.0.0",
-    "eslint-plugin-react": "^7.33.0",
-    "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-sort-destructure-keys": "^1.5.0",
-    "eslint-plugin-tailwindcss": "3.13.0",
-    "eslint-plugin-unicorn": "^48.0.1",
-    "prettier": "^3.0.0"
-  },
-  "engines": {
-    "node": ">=10"
```

Comment:
is there any restriction on node, eg only `>=20`? Asking for a few nodejs@16 friends :)

Reply:
Ah yep, good point. I thought it might make sense to remove this.

`eslint-config-molindo` is flexible about Node.js, it only requires ESM support via `.mjs` (which is supported [since Node.js 8.5](https://nodejs.org/api/esm.html)).

However, ESLint 9 [requires Node.js 18.8](https://github.com/eslint/eslint/blob/19e68d3f3a86cf23e5c98eaf8736eeaa33f194f4/packages/js/package.json#L29), so implicitly that is the lowest threshold. I thought removing this line might make sense, as we're essentially just repeating what ESLint has declared, and you'll already get a warning in older Node.js versions due to their `engines` setting.

For Node.js 16 apps, I guess it's ok to stay on ESLint 8 for the time being.

Does that sound reasonable to you?

Reply:
yes totally fine - thanks for clarifying.